tests/test_backend.py

```diff
@@ -298,11 +298,15 @@ def test_timeout_tiny(self):
 
     def test_set_add(self):
         self.cache.set("add_key", "Initial value")
-        self.cache.add("add_key", "New value")
-        res = cache.get("add_key")
+        res = self.cache.add("add_key", "New value")
+        self.assertEqual(res, False)
```

Comment:
I would use `assertIs(res, False)` to ensure res is actually false and not just falsey. Same for `assertIs(res, True)`.

From the [Python docs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue):

>  assertTrue(expr, msg=None)
> assertFalse(expr, msg=None)
>
> Note that this is equivalent to `bool(expr) is True` and not to `expr is True` (use `assertIs(expr, True)` for the latter).

Reply:
Whoops. I must have misread your initial comment. Thanks for your patience and kind clarification.